pkg/docker/docker.go

```diff
@@ -18,8 +18,8 @@ const (
 	ScriptsURLEnvironment = "STI_SCRIPTS_URL"
 	LocationEnvironment   = "STI_LOCATION"
 
-	ScriptsURLLabel = "io.openshift.sti.scripts-url"
-	LocationLabel   = "io.openshift.sti.location"
+	ScriptsURLLabel = "io.s2i.scripts-url"
+	LocationLabel   = "io.s2i.location"
```

Comment:
i've never liked calling this location... how about destination?  tmpdir?  target?  workdir?